lib/main_common.dart

```diff
@@ -41,19 +41,17 @@ class MyApp extends StatelessWidget {
             theme: ThemeData(
```

Comment:
Do we need this if we have the carriage theme?

Reply:
textTheme can be removed once everything is updated into the design system, but we'll still need the colors in the rest of the ThemeData